resources/views/blade/icon.blade.php

```diff
@@ -0,0 +1,10 @@
+{{-- IMPORTANT!!! Make sure there is no newline at the end of this file, or it will break the loaders for the tables --}}
+
+@props([
+    'type' => '',
+    'class' => false,
+    'style' => false,
+    'id' => false,
+    'title' => false,
+])
+<i {{ $attributes->merge(['class' => Icon::icon($type).' '.$class]) }} {{ isset($style) ? $attributes->merge(['style' => $style]): '' }}  {{ isset($title) ? $attributes->merge(['title' => $title]): '' }}  aria-hidden="true"></i>
```

Comment:
Another note: I think we should avoid using `isset` in components since that should be handled by `@props`. If something is in `@props` without a default value then the component will throw an exception when trying to render and if it does have a value then we don't need to use `isset` (we could add a conditional around the value if needed of course).

In this case we should be able to remove `class` from the props list since `{{ $attributes->class(...) }}` [should handle that](https://laravel.com/docs/10.x/blade#default-merged-attributes).

Reply:
I tried that, but it still threw an error that the variable was not set.